server/cmd/dcrdex/evm-protocol-overrides.json

```diff
@@ -0,0 +1,8 @@
+{
+    "eth": 0,
+    "usdc.eth": 0,
+    "polygon": 0,
+    "usdc.polygon": 0,
+    "wbtc.polygon": 0,
+    "weth.polygon": 0
```

Comment:
This is annoying, but until we launch contracts, you'll have to delete these overrides to test v1 contracts.